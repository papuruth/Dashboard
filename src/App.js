import React from 'react';
import { Route, Router, Switch } from 'react-router';
import ErrorBoundary from './utils/generalComponents/ErrorBoundary/index';
import history from './utils/routes/history';
import routes from './utils/routes/index';

function App() {
  return (
    <div>
      <Router history={history}>
        <ErrorBoundary>
          <Switch>
            {routes.map((item) => (
              <Route {...item} />
            ))}
          </Switch>
        </ErrorBoundary>
      </Router>
    </div>
  );
}

export default App;
