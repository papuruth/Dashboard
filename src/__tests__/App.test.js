import App from '@/App';
import { render } from '@testing-library/react';

describe('testing App.js component', () => {
  test('should mount app component', () => {
    const wrapper = render(<App />);
    expect(wrapper).toMatchSnapshot()
  });
});
