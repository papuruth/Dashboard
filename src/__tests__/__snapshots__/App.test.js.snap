// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing App.js component should mount app component 1`] = `<App />`;
